src/commands/synthetics/reporters/junit.ts

```diff
@@ -509,6 +543,10 @@ export class JUnitReporter implements Reporter {
   }
 
   private getTestCaseStats(result: Result): TestCaseStats {
+    if (isResultSkippedBySelectiveRerun(result)) {
+      return getDefaultTestCaseStats()
+    }
```

Comment:
This is also to align with `getSkippedTestCase()`:

https://github.com/DataDog/datadog-ci/blob/1d8e7821d132076a3237dc86f3bb185a355f3814/src/commands/synthetics/reporters/junit.ts#L439-L447